src/plugins/recordTypes/organization/vocabularies.js

```diff
@@ -0,0 +1,22 @@
+import { defineMessages } from 'react-intl';
+
+export default {
+  committee: {
+    messages: defineMessages({
+      name: {
+        id: 'vocab.organization.committee.name',
+        description: 'The name of the vocabulary.',
+        defaultMessage: 'Committee',
+      },
+      collectionName: {
+        id: 'vocab.organization.committee.collectionName',
+        description: 'The name of a collection of records from the vocabulary.',
+        defaultMessage: 'Committees',
+      },
+    }),
+    serviceConfig: {
+      servicePath: 'urn:cspace:name(committee)',
+    },
+    sortOrder: 0,
```

Comment:
I think we want to keep the same ordering of vocabularies as in 4.1 (corporate names before committees), so remove sortOrder.